Dockerfile

```diff
@@ -3,11 +3,13 @@
 # Base image to extend from
 FROM pegasystems/tomcat:9-jdk11 as release
 
+ARG VERSION
+
 LABEL vendor="Pegasystems Inc." \
       name="Pega Tomcat Node" \
-      version="2.1.0"
+      version=${VERSION:-CUSTOM_BUILD}
```

Comment:
Where do we fetch the value for CUSTOM_BUILD? Did we configure it in the repository settings?

Reply:
VERSION comes in as a build arg.  If that argument is not present (ie, someone made a local copy and are building without travis, they will build with the version string "CUSTOM_BUILD".  This will help differentiate between a CI/CD'd build and a local build with unknown changes in play.